documentation/docs/release-notes/3.2.0.md

```diff
@@ -23,6 +23,11 @@ PMM 3.2.0 introduces official RPM package support for installing the PMM Client
 
 This streamlines the setup process and integrates more seamlessly with your AL2023 environment, making it quicker and easier to get started with monitoring databases.
 
+### New dashboard for Percona Backup for MongoDB (PBM)
```

Comment:
It would be nice to have a dedicated dashboard page (linked here).
We can consider to mention the dashboard limitations and that we plan to improve in future releases - ref PMM-13752.
If not possible to have a dedicated page for this release, I would recommend to consider a doc ticket, which can be worked at some point.
Thanks